.github/workflows/pr.yml

```diff
@@ -29,9 +29,21 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           run_install: true
-      - run: pnpm build
       - run: pnpm lint
 
+  check:
+    name: Check 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - uses: pnpm/action-setup@v2
+        with:
+          run_install: true
+      - run: pnpm check
```

Comment:
If I understand correctly, `check` is like `build` but does not generate output. 
If that's the case, do we also need to run `build` in this workflow?

Reply:
No, it's not needed! The `check` task doesn't need to use the build process.

Reply:
Then can we remove [this job](https://github.com/pagopa/interop-be-monorepo/pull/460/files#diff-15c806aa509538190832852f439e9921a23bec2da81f95ed0e4bf13c14e5b160R60)? 🤔